sros2/verb/create_key.py

```diff
@@ -0,0 +1,34 @@
+# Copyright 2016-2017 Open Source Robotics Foundation, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+try:
+    from argcomplete.completers import DirectoriesCompleter
+except ImportError:
+    DirectoriesCompleter = None
+
+from ros2pkg.verb import VerbExtension
+from sros2.api import create_key
+
+
+class CreateKeyVerb(VerbExtension):
+    """Create key."""
+
+    def add_arguments(self, parser, cli_name):
+        arg = parser.add_argument('ROOT', help='root path of keystore')
+        arg.completer = DirectoriesCompleter()
```

Comment:
actually even if I have it installed it crashes. DirectoriesCompleter has been added in v0.8.5 and xenial ships with 0.8.1 (yeah I know zesty ships with 1.7 and pip with 1.8...) so I think the check on the import rule should be stricter